includes/class-wc-payments-account.php

```diff
@@ -1885,7 +1885,7 @@ public function is_card_testing_protection_eligible(): bool {
 	}
 
 	/**
-	 * Redirects to the onboarding flow page if the Progressive Onboarding feature flag is enabled or in the experiment treatment mode.
+	 * Redirects to the onboarding flow page if the Progressive Onboarding is enabled (default experience).
```

Comment:
We can probably simplify this even further to just `Redirects to the onboarding flow page.` since it's the default experience now - WDYT? 🙂

Reply:
Totally! Good point 👍

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/7773/commits/6b4b3d4536b8fa13538dda3b62e7f76ad0f469b2